runelite-client/src/main/java/net/runelite/client/plugins/fairyring/FairyRingPlugin.java

```diff
@@ -212,17 +239,17 @@ public void onGameTick(GameTick t)
 		// This has to happen because the only widget that gets hidden is the tli one
 		Widget fairyRingTeleportButton = client.getWidget(ComponentID.FAIRY_RING_TELEPORT_BUTTON);
 		boolean fairyRingWidgetOpen = fairyRingTeleportButton != null && !fairyRingTeleportButton.isHidden();
-		boolean chatboxOpen = searchInput != null && chatboxPanelManager.getCurrentInput() == searchInput;
+		boolean searchInputBoxOpen = searchInput != null && chatboxPanelManager.getCurrentInput() == searchInput;
+		boolean tagInputBoxOpen = tagInput != null && chatboxPanelManager.getCurrentInput() == tagInput;
 
-		if (!fairyRingWidgetOpen && chatboxOpen)
+		if (!fairyRingWidgetOpen && (searchInputBoxOpen || tagInputBoxOpen))
 		{
 			chatboxPanelManager.close();
 		}
 	}
 
-	private void updateFilter(String filter)
+	private void updateFilter(final String filter)
 	{
-		filter = filter.toLowerCase();
```

Comment:
It seems like all callsites are passing strings which are manually lowercased. Is there some reason for this change besides being able to make the parameter to be `final`?

Reply:
I think in one of the commits I made made it so that filter had to be final, I think I can roll that back now if you'd like.